### DISCUSSION TITLE
Simple way to get the best scores at the end of a training?

### DISCUSSION THREAD

**Discussion by jgsch** [Lightning-AI/pytorch-lightning#5817] [Lightning Trainer API: Trainer, LightningModule, LightningDataModule] [answered]
### What is your question?    
After a training, is there an easy way to get the best scores returned by the `validation_end` function? In order to use a hyperparameters optimizer like [Tune](https://ray.readthedocs.io/en/latest/tune.html).

#### Code example
```
model = CoolSystem()
trainer = Trainer()    
trainer.fit(model)   

best_scores = ???
print(best_scores)
```

#### What's your environment?   
- conda 4.7.10   
- PyTorch 1.3.0   
- Lightning 0.5.3.1

**Comment by williamFalcon** [comment, marked as answer]
Some options:
- print the scores in training_end
- log them to the logger
- access trainer.tqdm_metrics
